build.gradle

```diff
@@ -15,7 +15,6 @@ buildscript {
     dependencies {
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:${kotlin_version}"
         classpath "net.corda.plugins:cordapp:${corda_gradle_plugins_version}"
-        classpath "net.corda.plugins:cordformation:${corda_gradle_plugins_version}"
         classpath "net.corda.plugins:quasar-utils:${corda_gradle_plugins_version}"
```

Comment:
Do we need remove it too?

Reply:
And in repositories too

Reply:
This is general import of plugins for all projects, i think it should stay in one place.

Reply:
Conversation above